### ISSUE TITLE
ios端播放出现崩溃IJKSDLGLView displayInternal 、IJKSDLGLView display 次数很频繁 急 急 急

### ISSUE THREAD

**Issue by 976996968** [ksvc/KSYMediaPlayer_iOS#93]
![2018-02-08 10 49 40](https://user-images.githubusercontent.com/31341993/35953066-d423a98c-0cbd-11e8-9eab-f238c02bf85a.png)

**Comment by mayudong1** [comment]
这个问题一直都存在，查到过部分原因，但依然顽固的存在，暂时也没有找到出现这个崩溃的原因
所以暂时没有解决方案  

另：一直这个问题都是通过bugly之类的日志收集发现的，在测试过程中从来没有出现过，你这里能给提供会出现这个崩溃的场景吗？

**Comment by 976996968** [comment]
是的 一直都是收集崩溃日志发现的 之前跟你们提过这个 还是没有找到原因是吗？我现在收集到的好多日志都是类似这种ijk崩溃，我也没发现重现这个崩溃的场景，所以很难找问题

**Comment by mayudong1** [comment]
是的，顽疾了，以前找到过部分原因，但后来依然会出现，暂时也没有进一步的发现

**Comment by 976996968** [comment]
那这个怎么弄啊 崩溃日志好多这个 尝试过切后台或打电话来测试 也都没有复现 观看直播时除了这些也想不出有其他情况了 搞得头大

**Comment by poholo** [comment]
#119

**Comment by adamzbl** [comment]
出现频率较高，有一个场景是 在app后台时，调用 player -> pause ；暂停播放；APP在后台45分钟，返回时，调用 player -> play  [并且短时间内（1s内），销毁此player]；

我们经常遇到此问题基本都是在 销毁播放器时，代码顺序为 [ pause, stop, playerView removeFromsuperView]；

**Comment by adamzbl** [comment]
另外,IJK的git上貌似也有人提此问题，并且得到了修复，金山同学可以看下是否咱们分支有修复过同类问题。

**Comment by adamzbl** [comment]
* thread #1, queue = 'com.apple.main-thread'
    frame #0: 0x000000018504a7c8 libsystem_kernel.dylib`kevent_id + 8
    frame #1: 0x000000010498b03c libdispatch.dylib`_dispatch_kq_poll + 208
    frame #2: 0x000000010498a634 libdispatch.dylib`_dispatch_event_loop_poke + 428
    frame #3: 0x0000000187de239c FrontBoardServices`-[FBSWorkspaceClient _sendMessage:withEvent:] + 92
    frame #4: 0x0000000187de1e54 FrontBoardServices`-[FBSWorkspaceClient scene:didUpdateClientSettings:withDiff:transitionContext:] + 172
    frame #5: 0x0000000187dbeff8 FrontBoardServices`__59-[FBSSceneImpl updateClientSettings:withTransitionContext:]_block_invoke.178 + 36
    frame #6: 0x0000000187dbec44 FrontBoardServices`-[FBSSceneImpl updateClientSettings:withTransitionContext:] + 444
    frame #7: 0x0000000187dbf224 FrontBoardServices`-[FBSSceneImpl updateClientSettingsWithTransitionBlock:] + 100
    frame #8: 0x0000000187dbf178 FrontBoardServices`-[FBSSceneImpl updateClientSettingsWithBlock:] + 80
    frame #9: 0x000000018f190f68 UIKit`-[FBSScene(UIApp) updateUIClientSettingsWithBlock:] + 128
    frame #10: 0x000000018f56f5d8 UIKit`-[UIApplication _setIdleTimerDisabled:forReason:] + 212
    frame #11: 0x000000010181b0dc Ace`-[IJKMediaModule updateIdleTimer] + 148
    frame #12: 0x00000001018169c4 Ace`-[IJKFFMoviePlayerController setScreenOn:] + 64
    frame #13: 0x0000000101817b6c Ace`-[IJKFFMoviePlayerController stop] + 52
  * frame #14: 0x00000001017f7e5c Ace`-[KSYMoviePlayerController stop] + 172
